### DISCUSSION TITLE
How to know the count of old properties when using manifold_set_properties from C bindings?

### DISCUSSION THREAD

**Discussion by NickUfer** [elalish/manifold#1075] [General]
I'm currently trying to implement the rust bindings for the C function `manifold_set_properties`:

https://github.com/elalish/manifold/blob/bc08d41dc058908e3a4a2c33934049e71a258652/bindings/c/include/manifold/manifoldc.h#L206-L210

I think this is not correctly implementable, as there is no binding for `Manifold::NumProp()` to get the count of old properties per vertex.

Or am I missing something?

**Comment by pca006132** [comment]
Yes, it seems that we missed this one. Nice find!

Would you like to open a PR to fix it?

**Reply by NickUfer** [reply]
Sure will do that later today 😄

**Comment by NickUfer** [comment]
This was fixed with PR https://github.com/elalish/manifold/pull/1078
